File: subroutines/selected/dependencies/subroutine parlsq dependencies/subroutine writeout_h.f
      subroutine writeout_h(dir,ip,fp,rms,message,iprint)
      real*8 dir(3),ip(3),fp(3),rms
      integer iprint,kprint
      character*60 message

c     don't waste time
      kprint=mod(iprint,10)
      if (kprint .eq. 0) return

 10   format(a6,1x,3(f12.6,1x))
 20   format('     RMS deviation: ',f11.6)
      if (message(1:5) .ne. '     ') write(*,*) '  '//message
      write(*,10) '     D',dir
      write(*,10) '     I',ip
      write(*,10) '     F',fp
      write(*,20) rms
      end

File: fortran to python conversion/dependencies/subroutine writeout_h.py
def writeout_h(dir, ip, fp, rms, message, iprint):
	"""
	Write header???
	Write summary of the set of points
	***
	Parameters:
	dir: direction vector
	ip: initial point
	fp: final point
	rms: root mean square
	"""
	kprint = mod(iprint,10)
	# don't waste time
	if (kprint == 0):
		return
	if (message[:5] != ""):
		print(message)
	print("D ",dir)
	print("I",ip)
	print("F,",fp)
	print("RMS deviation:",rms)